app/UrlGenerator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\App;
+
+use Symfony\Bridge\Twig\Extension\RoutingExtension;
+use WMDE\Fundraising\Frontend\Infrastructure\UrlGenerator as UrlGeneratorInterface;
+
+class UrlGenerator implements UrlGeneratorInterface {
```

Comment:
I spend 2 min being confoosed at UrlGeneratorInterface, not finding which commit changed that name. Class could be TwigBasedUrlGenerator (or SilexBasedUrlGenerator?) so the unexpected aliasing is not needed.